agent/containermetadata/utils.go

```diff
@@ -58,7 +58,7 @@ func getMetadataFilePath(taskARN string, containerName string, dataDir string) (
 func getTaskMetadataDir(taskARN string, dataDir string) (string, error) {
 	taskID, err := getTaskIDfromARN(taskARN)
 	if err != nil {
-		fmt.Errorf("get task metadata directory: %v", err)
+		_ = fmt.Errorf("get task metadata directory: %v", err)
```

Comment:
This should probably return an error or be logged, not sure which one.  We don't want to just assign it to `_`.

Reply:
this seems like a bug. seems like we should be bubbling it up.